packages/vite/package.json

```diff
@@ -10,7 +10,7 @@
   },
   "peerDependencies": {
     "@embroider/core": "workspace:^",
-    "vite": "^5.2.0"
+    "vite": "^6.0.3"
```

Comment:
Shouldn't we allow both versions?

Reply:
counter point: is there any need to drop support for 5 🤷 We should be expanding the range because we don't have any functionality that requires v6

Reply:
no